index.html

```diff
@@ -0,0 +1,1447 @@
+
+    <title>Newsweek</title>
+  </head>
+
+
+  <body>
+
+<div class="header-screen d-flex">
+        
+<div class="header-left no-big">
+<img src="images/header.png">
+</div>
+
+
+<div class="header-right d-flex no-big">
+
+<div class="header-right-l">
+</div>
+<div class="header-right-r d-flex justify-content-center">
+        <h2 class="signin-h no-big">SIGN IN</h2>
+        <h2 class="subscribe-h no-big">SUBSCRIBE</h2>
+        <div class="bars-h no-big"><i class="fas fa-bars"></i></div>
+</div>
+
+</div>
+</div>
+
+    <div class = "container-fluid first">
+  <header class="text-center">
+<div class="date-h no-small">Thu, May 23, 2019 </div>
+        <img class="header-img no-small" alt="header" src="images/header.png">
+
+        <div class="btn-h-big no-small d-flex">
+                <h2 class="signbh no-small">SIGN UP</h2><h2 class="subsbh no-small">SUBSCRIBE ></h2>
+        </div>
+
+
+
+  </header>
+
+  <navbar class="navbar navbar-expand-md no-small">
+
+                <a class="navbar-brand" href="#">U.S.</a>
+                <a class="navbar-brand" href="#">World</a>
+                <a class="navbar-brand" href="#">Business</a>
+                <a class="navbar-brand" href="#">Tech & Science</a>
+                <a class="navbar-brand" href="#">Culture</a>
+                <a class="navbar-brand" href="#">Sports</a>
+                <a class="navbar-brand" href="#">Health</a>
+                <a class="navbar-brand" href="#">Opinion</a>
+                <a class="navbar-brand" href="#">Ndelible</a>
+
+                <form class="form-inline my-2 my-lg-0">
+                                <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
+                                <button class="btn btn-light my-2 my-sm-0" type="submit"><i class="fas fa-search"></i></button>
+                              </form>
+
+
+</navbar>
+    </div>
+
+
+<!-- ***** STARTS 2TH COLUMN ****** --->
+
+
+ <!-- 2 - Second - Top Story ¡--->
+<div class="margin-con first-con-marg">
+<div class="row no-gutters more-pad">
+    <div class= "container-fluid second col-md-7 col-lg-5 order-md-2">
+<h2 class="headings-bigc">TOP STORY        </h2>
+    <div class="box2">
+
+            <div class="box2-up">
+
+                <h2 class="tags text-center">U.S.</h2>
+            </div>
+
+            <div class="box2-mid">
+                    <h2>‘You Take Care of Your Own’: Ben Carson Defends HUD Plan to Evict 55K Kids</h2>
+                    <p>Carson compared his HUD plan to flight attendants telling passengers to put on their oxygen masks before helping others.</p>
+            </div>
+
+            <div class="box2-down">
+
+                <h2>HUD Official Files Complaint After Demotion </h2>
+            </div>
+
+
+    </div>
+
+
+<!-- 3 - Third - Featured Stories ¡--->
```

Comment:
extra - in your comment (not necessary to remove but recommended, if choose to remove address for all instances of this warning.

Reply:
corrected in the last commit